includes/bootstrap.php

```diff
@@ -19,7 +19,7 @@
 
 // Version of the plugin.
 if ( ! defined( 'WP_ABILITIES_API_VERSION' ) ) {
-	define( 'WP_ABILITIES_API_VERSION', '0.2.0' );
+	define( 'WP_ABILITIES_API_VERSION', '0.3.0' );
```

Comment:
This deserves its own call out. This PR is focused on adding a categories system, not cutting a new release.

```suggestion
	define( 'WP_ABILITIES_API_VERSION', '0.2.0' );
```

Reply:
Good note. Versioning should be done in a subsequent PR.

Reply:
Done